handlers/handlers.go

```diff
@@ -202,3 +214,110 @@ func ListPageRender(rend RenderClient, cli CodeListClient) http.HandlerFunc {
 		return
 	}
 }
+
+//AreaPageRender ...
```

Comment:
I had a similar comment on my PR - it's probably worth completing this comment so that it explains what the func does.